src/main/java/games/strategy/triplea/ai/proAI/ProTechAI.java

```diff
@@ -602,20 +584,18 @@ private static Route getMaxSeaRoute(final GameData data, final Territory start,
   }
 
   /**
-   * Gets the neighbors which are exactly a certain # of territories away (distance)
-   * Removes the inner circle neighbors
-   * neutral - whether to include neutral countries.
+   * Gets the neighbors which are one territory away.
    */
-  private static List<Territory> getExactNeighbors(final Territory territory, final int distance, final GameData data,
-      final boolean neutral) {
+  private static List<Territory> getExactNeighbors(final Territory territory, final GameData data) {
```

Comment:
Funny...  Now that it's clear what this method is doing, we probably could have forgone the Javadocs entirely and just renamed it `getAdjacentTerritories()` or something similar.  NBD.